wherehows-web/app/components/dataset-compliance.js

```diff
@@ -7,14 +7,21 @@ const {
   get,
   isBlank,
   setProperties,
-  getWithDefault
+  getWithDefault,
+  String: { htmlSafe }
 } = Ember;
 
 // TODO: DSS-6671 Extract to constants module
 const missingTypes = 'Looks like some fields may contain privacy data ' +
   'but do not have a specified `Field Format`?';
 const successUpdating = 'Your changes have been successfully saved!';
 const failedUpdating = 'Oops! We are having trouble updating this dataset at the moment.';
+const hiddenTrackingFieldsMsg = htmlSafe(
```

Comment:
Last time we have html inside JavaScript in TREX, we were told that it's not recommended and that person raised a lot of open issue in our RB. You can try this, but maybe consider using `Ember partial helper ` instead. And it's easy to maintain too.